package.json

```diff
@@ -16,10 +16,6 @@
     "url": "https://github.com/winjs/react-winjs"
   },
   "license": "MIT",
-  "peerDependencies": {
-    "react": "0.13.x",
-    "winjs": "4.4.x"
-  },
```

Comment:
Why remove `peerDependencies`? It signals that this package expects those other dependencies to be installed alongside.